Sources/Sentry/SentryDependencyContainer.m

```diff
@@ -44,24 +44,30 @@ + (void)initialize
 {
     if (self == [SentryDependencyContainer class]) {
         sentryDependencyContainerLock = [[NSObject alloc] init];
+        instance = [[SentryDependencyContainer alloc] init];
```

Comment:
The next step would be to initialize the `instance` when SentrySDK is started. This approach allows us to pass the options and eliminate any dependencies on `SentryDependencyContainer` from both the hub and client, resolving the unusual cyclic dependency.

Reply:
Yes, I agree with that approach 👍🏻